README.md

```diff
@@ -85,16 +87,12 @@ let provider = ethers.getDefaultProvider();
 // other context like wallet, signer etc all can be passed in as well.
 const multicall = new Multicall({ ethersProvider: provider, tryAggregate: true });
 
-const contractCallContext: ContractCallContext<{extraContext: string, foo4: boolean}>[] = [
+const contractCallContext: ContractCallContext[] = [
```

Comment:
Keep this example with extra context it just shows how it should work if you want to pass some extra context in (it does work I use it on a few places)

Reply:
I use it as well :)

My logic here is that there is already a section of the readme called `passing extra context to the call`, where this example is shown.

On the section I removed it in, the extra context is added, but the resulting example does not show the extra context that was passed. I don't believe it was intended to add that context when the documentation was written.

Reply:
Ahhhh ok good spot keep it then thanks for that!